app/assets/javascripts/dashboard.js.coffee

```diff
@@ -23,8 +23,6 @@ channelBind = (userChannel) ->
   return
 
 initialize = ->
-  pusher = new Pusher(pusher_key)
-  userChannel = pusher.subscribe('dashboard_channel')
   channelBind(userChannel)
```

Comment:
I would leave this behavior here and put some variable with debate id in dashboard template